WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -295,6 +324,14 @@ class ReaderPostListViewModel @Inject constructor(
         isFirstLoad = false
     }
 
+    fun onSubfilterChanged(subfilterListItem: SubfilterListItem) {
+        changeSubfilter(subfilterListItem, true)
+    }
+
+    fun manageSubfilter() {
+        changeSubfilter(getCurrentSubfilterValue(), false)
+    }
```

Comment:
It took me a while to understand what `manageSubfilter()` actually does and I think a more descriptive name could help.

How about renaming these functions to something like:
-  `onSubfilterChanged` to `onSubfilterSelected`
-  `manageSubfilter ` to `onSubfilterReselected`

Reply:
Makes sense. Changed it [here](09f76fb894bbe9e6dc16f762f0d44e01d16c4ae8) and [here](ac146f1ff94197769160de6644e4c2a3d16de2c6)  👍